include/d/t/d_t_heat_resist.h

```diff
@@ -1,14 +1,23 @@
 #ifndef D_T_HEAT_RESIST_H
 #define D_T_HEAT_RESIST_H
 
-#include "d/a/d_a_base.h"
+#include "common.h"
+#include "d/t/d_tg.h"
+#include "m/m_mtx.h"
 
-class dTgHeatResist_c : public dAcBase_c {
+class dTgHeatResist_c : public dTg_c {
 public:
     dTgHeatResist_c() {}
     virtual ~dTgHeatResist_c() {}
 
+    virtual int create() override;
+    virtual int doDelete() override;
+    virtual int draw() override;
+    virtual int actorExecute() override;
+
 private:
+    mMtx_c matrix;
+    u8 mInverted;
```

Comment:
we like to add prefix comments to signify the offset of the variables
```suggestion
    /* 0x0FC */ mMtx_c matrix;
    /* 0x12C */ u8 mInverted;
```

Reply:
Thank you, I will take it into account